admin-dev/themes/new-theme/js/pages/product/components/create-product-modal.ts

```diff
@@ -40,29 +50,35 @@ export default class CreateProductModal {
       const $link = $(event.target);
       const linkUrl = `${$link.prop('href')}&liteDisplaying=1`;
 
-      const iframeModal = new FormIframeModal({
-        id: 'modal-create-product',
-        formSelector: 'form[name="product"]',
-        formUrl: linkUrl,
-        closable: true,
-        // We override the body selector so that the modal keeps the size of the initial create form even after submit
-        autoSizeContainer: '.create-product-form',
-        onFormLoaded: (form: HTMLElement, formData: FormData, dataAttributes: DOMStringMap | null): void => {
-          if (dataAttributes) {
-            if (dataAttributes.modalTitle) {
-              iframeModal.setTitle(dataAttributes.modalTitle);
-            }
+      this.openCreationModal(linkUrl);
+    });
+  }
+
+  private openCreationModal(linkUrl: string): void {
+    const iframeModal = new FormIframeModal({
+      id: ProductMap.create.modalId,
+      // We use the edit form selector because it's the one we need to get data attributes from
+      formSelector: ProductMap.productForm,
+      formUrl: linkUrl,
```

Comment:
following above comment this probably could become
```suggestion
      formUrl,
```